pkgs/applications/editors/emacs/elisp-packages/manual-packages/voicemacs/default.nix

```diff
@@ -44,6 +45,8 @@ melpaBuild {
     yasnippet
   ];
 
+  passthru.updateScript = unstableGitUpdater { hardcodeZeroVersion = true; };
```

Comment:
I suggest that we do not add a update script to packages without a maintainer.  Auto bump PRs need people to test.

Reply:
I will adopt them, then.

https://github.com/NixOS/nixpkgs/pull/329248
https://github.com/NixOS/nixpkgs/pull/329249